### PR TITLE
Different Bug Fixes

### PR DESCRIPTION
**Pull request by MartinBruun**

### CODE REVIEW

**Comment by MartinBruun** [comment]
Makes it now present correct information in CalendarView
Makes the Account now work more correctly (does not always post to account 1 for instance...)

BUGS NEEDED TO FIX: It is possible to delete active bookings (only fetch non-activated bookings)
BUGS NEEDED TO FIX: Currently fetches ALL bookings, not ones OWN bookings!!!
More to be found tommorow!

File: washee_app/lib/features/booking/presentation/widgets/day_card.dart [hunk 1]
```diff
@@ -43,18 +43,23 @@ class _DayCardState extends State<DayCard> {
...
   DateHelper helper = DateHelper();
 
   @override
   void initState() {
     var calendar = Provider.of<CalendarProvider>(context, listen: false);
     _isToday = helper.isToday(widget.currentDate);
-    _bookingsForCurrentDay = calendar.getBookingsForDay(widget.currentDate);
-    _numberOfOccupiedSlots = calendar.getNumberOfOccupiedSlots(
-        _bookingsForCurrentDay, widget.currentDate);
+    _washBookingsForCurrentDay = calendar.getBookingsForDay(widget.currentDate, 1);
```
Comment (Rallefar1103) [inline washee_app/lib/features/booking/presentation/widgets/day_card.dart:59]: brug gerne MachineType.Washmachine og MachineType.Dryer

File: washee_app/lib/features/booking/presentation/widgets/day_card.dart [hunk 2]
```diff
@@ -126,16 +128,26 @@ class _DayCardState extends State<DayCard> {
...
-            context: context,
-            builder: (BuildContext context) {
-              return ChooseMachineView(
-                washesForDay: _washBookingsForCurrentDay,
-                dryingsForDay: _dryBookingsForCurrentDay,
-                currentDate: widget.currentDate,
-              );
-            },
-          );
+          ActiveUser user = ActiveUser();
```
Comment (Rallefar1103) [inline washee_app/lib/features/booking/presentation/widgets/day_card.dart:131]: det her er underligt. Hvorfor bliver den instantieret her? Bliver det ikke gjort globalt tidligere?

**Review by Rallefar1103 — COMMENTED** [review]
se mine små kommentarer

**Review by Rallefar1103 — APPROVED** [review]

**Pull request merged**